src/Components/Container.php

```diff
@@ -95,11 +97,30 @@ public function getOperation($need = TRUE)
     /**
      * Returns export component.
      * @param bool $need
-     * @return Export
+     * @return CsvExport
+     *
+     * @deprecated
      */
     public function getExport($need = TRUE)
```

Comment:
What about rename parameter and inside decide if it's bool or string. In case of string (as identifier of export) it should return that export. In case of bool warning about deprecated.